run_tests.sh

```diff
@@ -1,11 +1,22 @@
 #!/usr/bin/env bash
 
-images=("php" "fpm" "apache-php" "akeneo-apache")
-versions=("5.6" "5.6" "php-5.6" "php-5.6")
+didFail=
+images=("php" "fpm" "apache-php")
+versions=("5.6" "php-5.6" "php-5.6")
 
 cwd=$(pwd)
 for (( i=0; i<${#images[@]}; i++ ));
 do
     cd ${cwd}/${images[i]}
     docker build -t akeneo/${images[i]}:${versions[i]} .
+
+    if ! docker inspect akeneo/${images[i]}:${versions[i]} &> /dev/null; then
+		echo $'\timage does not exist!'
+		didFail=1
+		continue
+    fi
 done
+
+if [ "$didFail" ]; then
```

Comment:
or you could maybe directly do `return "$didFail"` and initialize didFail with 0